### DISCUSSION TITLE
Recordings tab

### DISCUSSION THREAD

**Discussion by rbg33** [Ylianst/MeshCentral#5607] [Q&A] [answered]
Hello!
I need to set up MeshCentral with the ability to record video and the Recordings tab like in this video
https://youtu.be/nDkNbYE8fT0?t=287

The first version I see on github is 1.0.44 (2022), but it no longer has this tab. How can I achieve the desired result?

**Comment by si458** [comment, marked as answer]
you simple enable it in your `config.json` and put the following inside your `domains ""` array, then restart meshcentral
```
      "sessionRecording": {
        "protocols": [1,2,5,100,101,200],
        "index": false
      },
```

**Reply by rbg33** [reply]
With these settings, the recording tab does not appear.

Is everything correct? Thank you.

![Screenshot_114](https://github.com/Ylianst/MeshCentral/assets/151945611/ba2735e8-4e27-4465-a627-9189a83da88d)

**Reply by si458** [reply]
https://ylianst.github.io/MeshCentral/meshcentral/faq/#json-config-files
you have it commented out so its going to be ignored, you need to remove the `_` for every entry you want it to use

note: what each setting means and if you need them or not compared to defaults
![image](https://github.com/Ylianst/MeshCentral/assets/765314/f4266a5e-bed5-4ca6-925b-1325a517107f)

**Reply by rbg33** [reply]
Is there something wrong?

**### error**
![Screenshot_117](https://github.com/Ylianst/MeshCentral/assets/151945611/ab40d277-ac69-4b02-b3e5-e098027d02fc)

### **config**
![Screenshot_115](https://github.com/Ylianst/MeshCentral/assets/151945611/d3638954-c9a9-435c-867e-5a27c41f9c99)

### **mesherrors**
![Screenshot_116](https://github.com/Ylianst/MeshCentral/assets/151945611/bda67ea7-0931-4daf-95aa-b085d13146c4)

**Reply by si458** [reply]
yes, sessionRecording should be INSIDE the "" of domains, so it should be 
```
"domains": {
  "": {
    "title": "mytitle",
    "title2": "mytitle2"
    ...
   "sessionRecording": {
      "protocols": [ 1,2,5,100,101,200]
      ..
  }
}
```

**Reply by rbg33** [reply]
Thank you!!!
